lsmb-request.pl

```diff
@@ -19,7 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
-
+use lib '.';
```

Comment:
Actually, that exact solution won't quite work if the file isn't in the TL dir of LedgerSMB, rather we would want to strip off the trailing portion of the path, so we end up with the correct parent.